mediator/med_map_mod.F90

```diff
@@ -36,14 +35,23 @@ module med_map_mod
 
   ! public routines
   public :: med_map_RouteHandles_init
+  public :: med_map_RH_is_created
   public :: med_map_Fractions_init
   public :: med_map_MapNorm_init
   public :: med_map_FB_Regrid_Norm
-  public :: med_map_uv_cart3d
+  public :: med_map_FB_Field_Regrid
+  public :: med_map_Field_Regrid
 
   interface med_map_FB_Regrid_norm
      module procedure med_map_FB_Regrid_Norm_All
+#if (1 == 2)
```

Comment:
I'd like to confirm with @mvertens that we can remove the med_map_FB_Regrid_Norm_Frac method.  I cpp'ed it out (as above) and have reviewed the mediator, it's not needed there.  But I want to make sure it's not used somewhere else in CESM or otherwise or that we don't want to keep it for future reference.  I agree that we either need to update the method (to make it consistent with other changes) or remove it.

Reply:
@apcraig @jedwards4b  - I am fine with removing this method.